msgraph-developer-proxy/appsettings.json

```diff
@@ -12,5 +44,12 @@
     "https://*.sharepoint.*/*_vti_bin/*",
     "https://*.sharepoint-df.*/*_api/*",
     "https://*.sharepoint-df.*/*_vti_bin/*"
-  ]
+  ],
+  "mocksPlugin": {
+    "mocksFile":  "mocks.json"
+  },
+  "randomErrorsPlugin": {
+    "rate": 99,
```

Comment:
Since this is a default value for all users, shall we leave it at the default 50 which we have right now?